media-fonts/intel_one_mono/intel_one_mono-1.4.0.recipe

```diff
@@ -0,0 +1,36 @@
+SUMMARY="A monospaced font family built with the needs of developers in mind"
+DESCRIPTION="Introducing Intel One Mono, an expressive monospaced font family that’s built with \
+clarity, legibility, and the needs of developers in mind.
+
+It’s easier to read, and available for free, with an open-source font license.
+
+Identifying the typographically underserved low-vision developer audience, Frere-Jones Type \
+designed the Intel One Mono typeface in partnership with the Intel Brand Team and VMLY&R, for \
+maximum legibility to address developers' fatigue and eyestrain and reduce coding errors. A \
+panel of low-vision and legally blind developers provided feedback at each stage of design.
+
+Intel One Mono also covers a wide range of over 200 languages using the Latin script. The Intel \
+One Mono fonts are provided in four weights — Light, Regular, Medium, and Bold — with matching \
+italics, and we are happy to share both an official release of fonts ready to use as well as \
+editable sources."
+HOMEPAGE="https://github.com/intel/intel-one-mono"
+COPYRIGHT="2023-2024 The Intel One Mono Project Authors"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/intel/intel-one-mono/releases/download/V$portVersion/otf.zip"
+CHECKSUM_SHA256="74ef8ee667403c760745bc12fc5e2cb1684544194fad3d5340919c173a8227fc"
+SOURCE_DIR=""
```

Comment:
Could you add `SOURCE_FILENAME="intel-one-mono-$portVersion.zip"` please? The download file name seems to be just "otf.zip", and it should contain the portVersion, otherwise updates will be problematic because it tries to reuse the download then.

Reply:
I missed that point also for a long time, but got bitten by it when upgrading texlive before, so I [added it in the last update](https://github.com/haikuports/haikuports/commit/068d977e576b2cb2f5a4db5f2e222c567c3be410#diff-1fc3b5e16bcfaccf1721278c92eca14616280ea9a4afb8a300588fb5ccd10e57R20). It is also part of the [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name) already.

Reply:
I would say the concrete file name doesn't matter that much, but it should at least contain the portVersion in some form, so that future updates will result in a different file name, just to make the life of packagers easier.

Reply:
This doesn't seem to have a version at all, so I would say no. If it gets updated upstream and the URL doesn't change, then that is problematic anyway.